CMakeLists.txt

```diff
@@ -20,104 +21,92 @@
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.8)
 project(gr-inspector CXX C)
 enable_testing()
 
-cmake_policy(SET CMP0015 NEW)
-
-#install to PyBOMBS target prefix if defined
+# Install to PyBOMBS target prefix if defined
 if(DEFINED ENV{PYBOMBS_PREFIX})
     set(CMAKE_INSTALL_PREFIX $ENV{PYBOMBS_PREFIX})
     message(STATUS "PyBOMBS installed GNU Radio. Setting CMAKE_INSTALL_PREFIX to $ENV{PYBOMBS_PREFIX}")
 endif()
 
-#select the release build type by default to get optimization flags
+# Select the release build type by default to get optimization flags
 if(NOT CMAKE_BUILD_TYPE)
    set(CMAKE_BUILD_TYPE "Release")
    message(STATUS "Build type not specified: defaulting to release.")
 endif(NOT CMAKE_BUILD_TYPE)
 set(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE} CACHE STRING "")
 
-#make sure our local CMake Modules path comes first
+# Make sure our local CMake Modules path comes first
 list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_SOURCE_DIR}/cmake/Modules)
 
 # Set the version information here
-set(VERSION_INFO_MAJOR_VERSION 1)
-set(VERSION_INFO_API_COMPAT    1)
-set(VERSION_INFO_MINOR_VERSION 2)
-set(VERSION_INFO_MAINT_VERSION git)
-
-# Set cmake policies.
-# This will suppress developer warnings during the cmake process that can occur
-# if a newer cmake version than the minimum is used.
+set(VERSION_MAJOR 1)
+set(VERSION_API    0)
+set(VERSION_ABI 0)
+set(VERSION_PATCH git)
```

Comment:
You are right, though I have not much experience with semantic versioning. Is there a GR guideline to stick to? Anyway I'm happy to bump to 2.0.0.

Reply:
Yes, there's a guideline [here](https://github.com/gnuradio/gnuradio/blob/master/VERSIONING), but it applies to the core codebase. However, it's based on [semver](https://semver.org/). 

So, here's where it get's tricky: If no functionality had changed, it'd say that the compatibility with GR 3.8 would have made an ABI only change, but given that there's some functionality taken out (AMC and visu of FAM Data), the API has also changed. By GR standards, the dev paradigm stays would, in my understanding, stay the same. Ergo, this PR would be an API bump. 

It'd like @marcusmueller input on that, though. I find the "versioning" topic very interesting but still have my doubts on assertive bumping from such big changes.

Reply:
Not sure I'd 100% agree on a Major bump here 😅

Reply:
Then I still don't get it. According to semver:
> Major version X (X.y.z | X > 0) MUST be incremented if any backwards incompatible changes are introduced to the public API. It MAY also include minor and patch level changes. Patch and minor version MUST be reset to 0 when major version is incremented.

Here, due to the removal of two blocks, we have a backwards incompatible change. What speaks against a major bump?

Reply:
You are right, I ignored the removal of the blocks when I wrote that comment two days ago. Then, a major bump makes more sense to me now.